test/D2L.SQL.UnitTests/ValidatorTests.cs

```diff
@@ -72,6 +72,32 @@ internal sealed class ValidatorTests {
 		[TestCase( "SELECT mt.HOST, nmt.HOST, mt.DOMAIN, nmt.DOMAIN FROM TEST_WEB_STAT_MULTI_TENANT mt INNER JOIN TEST_WEB_STAT nmt ON (mt.HOST = nmt.HOST) LIMIT 1" )]
 		[TestCase( "SELECT mt.HOST, nmt.HOST, mt.DOMAIN, nmt.DOMAIN FROM TEST_WEB_STAT_MULTI_TENANT mt, TEST_WEB_STAT nmt WHERE mt.HOST = nmt.HOST LIMIT 1" )]
 		[TestCase( "SELECT NAME.ALPHA, NAME.BETA, SOME_UIDS.ALPHA, SOME_UIDS.BETA, SOME_UIDS.NAME.TENANT, NAME.INTERACTIONTYPE FROM SOME_UIDS SOME_UIDS LIMIT 20" )]
+		// Tests for compound column names (Apache Phoenix SQL)
+		[TestCase( @"SELECT PHOENIX_TABLE.A.ID, PHOENIX_TABLE.A.V, PHOENIX_TABLE.A.TIMESTAMP, PHOENIX_TABLE.A.USERID, PHOENIX_TABLE.A.TYPE, PHOENIX_TABLE.PHONEID 
+					FROM PHOENIX_TABLE PHOENIX_TABLE ORDER BY PHOENIX_TABLE.A.USERID" )]
+		[TestCase( @"SELECT PHOENIX_TABLE.A.ID WHERECLAUSE, PHOENIX_TABLE.A.V, PHOENIX_TABLE.A.TIMESTAMP, PHOENIX_TABLE.A.USERID, 
+					PHOENIX_TABLE.A.TYPE, PHOENIX_TABLE.PHONEID 
+					FROM PHOENIX_TABLE PHOENIX_TABLE
+					WHERE (PHOENIX_TABLE.A.TIMESTAMP>'45') ORDER BY PHOENIX_TABLE.A.TIMESTAMP, PHOENIX_TABLE.A.ID" )]

```

Comment:
For `whereOpt.Rule = Empty | WHERE + expression;` - where is `expression` defined to use `column`?